source/_patterns/01-molecules/navigation/site-header-nav-bar~secondary.json

```diff
@@ -4,35 +4,24 @@
 
   "linkedItems": [
     {
-      "text": "Newsletters",
-      "path": "#",
-      "classes": "nav-secondary__item"
+      "text": "Search",
+      "rel": "search",
+      "classes": "nav-secondary__item nav-secondary__item--search",
```

Comment:
We will need to potentially adjust the ViewModel to all for a custom class in this object for each linkedItem

Reply:
Is there anything to be done about this comment? Can you add more details please?